hr_attendance_sheet/models/hr_attendance.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2020 Pavlov Media
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
```

Comment:
```suggestion
# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
```

I'm not sure this module is AGPL or LGPL?

Reply:
It's AGPL, I fixed all the references to LGPL and change it. Thanks.